### ISSUE TITLE
Skip client ID verification

### ISSUE THREAD

**Issue by rgarza** [okta/okta-jwt-verifier-golang#35]
Hello, is there a way to skip the client id verification? i just need to validate that it is indeed a valid Token for any client Id.

Thanks

**Comment by bretterer** [comment]
No, there is no way to skip it.  This library was designed for a specific purpose in mind.  I would suggest using a standard jwt library to do this.

**Comment by kmerfeld** [comment]
Can I ask what the specific purpose is?
I ran into the same issue so I just pulled the client id from the jwt and feed that into the jwtVerifier.
It doesn't seem like there is a technical reason to force checking the client id

**Comment by CarlosPadilla** [comment]
BTW, in the node library, the client Id is optional already, 
I think that would be OK as the library is able to check if the claims are present and then validate between the configuration and the JWT
It is not a big deal, is it?

**Comment by CarlosPadilla** [comment]
we did it it the code we are using, but now we are a few commits behind

https://github.com/okta/okta-jwt-verifier-golang/blob/7e1fd3a7929f35b5426c5d5df2d09603806ceec3/jwtverifier.go#L206

```go
if cid, ok := j.ClaimsToValidate["cid"]; ok && clientId != cid {
	return fmt.Errorf("clientId: %s does not match %s", clientId, j.ClaimsToValidate["cid"])
}
return nil
```

**Comment by bretterer** [comment]
This was merged in with #44
